updatecli/updatecli.d/update-container-image-tag.yaml

```diff
@@ -25,7 +26,38 @@ targets:
   updateValuesFile:
     kind: yaml
     sourceid: source/valuesFile
+    scmid: "default"
     spec:
       file: "charts/{{ requiredEnv .chart  }}/values.yaml"
       key: '{{ requiredEnv .containerImageTagValuePath }}'
       value: '{{ requiredEnv .releaseVersion }}'
+
+actions:
+  action/createUpdatePR:
+    helm-charts:
+    kind: "github/pullrequest"
+    scmid: "default"
+    spec:
+      automerge: true
+      mergemethod: squash
+      description: "Helm chart update"
+      draft: false
+      labels:
+        - "kind/enhancement"
+      title: "Helm chart update"
```

Comment:
I don't know if that helps, but updatecli can create a GH PR too. Maybe this would allow us to reduce the number of GH actions we depend upon

Reply:
@flavio That's what the "actions" do

On the updatecli side, we are currently deprecating `pullrequests` in favor of `actions`
In very very short, a pullrequest is github specific and we want to move the git logic for a target to a action.
More information on https://github.com/updatecli/updatecli/issues/465

You can run `updatecli manifest upgrade --config updatecli/updatecli.d` on your manifest to see what would change for your current updatecli version and `updatecli manifest upgrade --config updatecli/updatecli.d --in-place` to effectively upgrade your manifest if you are happy with the result.